localstack/mock/infra.py

```diff
@@ -35,6 +36,8 @@
 INSTALL_DIR_ES = '%s/elasticsearch' % INSTALL_DIR_INFRA
 TMP_ARCHIVE_ES = '/tmp/localstack.es.zip'
 
+ERROR_PROBABILITY = 0.05
```

Comment:
yea.. actually that is not implemented right now.  if we want to extend it to return errors randomly, it should be pretty straightforward